src/lib/lessons.ts

```diff
@@ -4,23 +4,49 @@ import getAccessTokenFromCookie from '../utils/get-access-token-from-cookie'
 import {loadLessonComments} from './lesson-comments'
 import {sanityClient} from 'utils/sanity-client'
 import groq from 'groq'
+import some from 'lodash/some'
+import isEmpty from 'lodash/isEmpty'
 
+// next_up_url can be derived on the front-end from collection and
```

Comment:
We aren't trying to handle `next_up_url` anymore. The lesson page already knows how to do that.

```suggestion
```